crlcore/python/technos/node180/gf180mcu/techno.py

```diff
@@ -79,21 +124,22 @@ def _setup_techno():
     CONT = createVia( tech, 'CONT_POLY2', 'Poly2', 'Contact', 'Metal1', u(0.22) )
     setEnclosures( CONT, Poly2 , u(0.07) )
     setEnclosures( CONT, Metal1, u(0.12) )
-    VIA12 = createVia( tech, 'VIA12', 'Metal1', 'Via1', 'Metal2', u(0.26) )
-    setEnclosures( VIA12, Metal1, (u(0.06), u(0.00)) )
-    setEnclosures( VIA12, Metal2, (u(0.06), u(0.01)) )
-    VIA23 = createVia( tech, 'VIA23', 'Metal2', 'Via2', 'Metal3', u(0.26) )
-    setEnclosures( VIA23, Metal2, (u(0.06), u(0.01)) )
-    setEnclosures( VIA23, Metal3, (u(0.01), u(0.06)) )
-    VIA34 = createVia( tech, 'VIA34', 'Metal3', 'Via3', 'Metal4', u(0.26) )
-    setEnclosures( VIA34, Metal3, (u(0.01), u(0.06)) )
-    setEnclosures( VIA34, Metal4, (u(0.06), u(0.01)) )
-    VIA45 = createVia( tech, 'VIA45', 'Metal4', 'Via4', 'Metal5', u(0.26) )
-    setEnclosures( VIA45, Metal4, (u(0.06), u(0.01)) )
-    setEnclosures( VIA45, Metal5, (u(0.01), u(0.06)) )
-    VIA5T = createVia( tech, 'VIA5T', 'Metal5', 'Via5', 'MetalTop', u(0.26) )
-    setEnclosures( VIA5T, Metal5  , (u(0.01), u(0.06)) )
-    setEnclosures( VIA5T, MetalTop, (u(0.06), u(0.01)) )
+    VIAS( useHV ).setup( tech )
```

Comment:
Maybe remove the commented code?

Reply:
Yes, I will. But let this one be commited !